fitbenchmarking/templates/css/table_style.css

```diff
@@ -38,10 +38,8 @@ th, td {
 }
 
 th a, td a {
-  display: table-row;
   color: rgb(30, 30, 30);
-  margin: -10em;
-  padding: 10em;
```

Comment:
Oh nevermind, now that the compare table is 2 anchors it doesn't work that way. That's fine.

Reply:
From some quick checks, `display: block` means we don't need this hack anymore! I suspect we were working around not knowing about `white-space: nowrap`

Reply:
I used `display: table-row;` in the last PR because the acc and runtime results were displayed compactly in the compare table. From what I remember ,`display: block;` introduced an extra line that did not look good.

<img width="907" alt="image" src="https://user-images.githubusercontent.com/47083562/229702949-57494fe0-5844-4009-bc3d-1e7fb590487b.png">

I was able to get the results I wanted using `white-space: nowrap` that is why I removed `display: table-row;` in this PR.

Reply:
With:
```
th a, td a {
  display: block;
  white-space: nowrap;
}
```
and no `<br>`

I see 
![image](https://user-images.githubusercontent.com/48282025/229732746-39de936a-c3f1-424d-b69e-9664289a4d8c.png)

And the area that you can click extends for the full cell